examples/localvideofilter/src/helpers-video-processor.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+var Video = require('twilio-video');
+
+/**
+ * Apply a filter to the frames of a LocalVideoTrack.
+ * @param {number} width - Width in pixels of the video frames
+ * @param {number} height - Height in pixels of the video frames
+ * @param {'blur' | 'grayscale' | 'sepia'} name - Filter name
+ * @param {string} settingValue - Filter setting value
+ * @constructor
+ */
+function FilterVideoProcessor(width, height, name, settingValue) {
+  this._outputFrame = new OffscreenCanvas(width, height);
+  this._outputContext = this._outputFrame.getContext('2d');
+  this._outputContext.filter = name + '(' + settingValue + ')';
```

Comment:
Since we're not doing anything else with these two parameters, can we just accept a `filter` parameter which is basically a [CanvasRenderingContext2D.filter](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/filter)?